File: Java/1008.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		Scanner scanner = new Scanner(System.in);
		
		int n = scanner.nextInt();
		int m =scanner.nextInt();
		float s = scanner.nextFloat();
		
        System.out.println(String.format("NUMBER = %d", n)); 
        System.out.println(String.format("SALARY = U$ %.2f", m * s)); 
	}
}

File: 1. Beginner/1008/1008.kt
import java.util.Scanner;

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`);
    
    val n: Int = scanner.nextInt();
    val m: Int = scanner.nextInt();
    val s: Float = scanner.nextFloat();
    
    println("NUMBER = %d\nSALARY = U$ %.2f".format(n, m * s))
}